### DISCUSSION TITLE
Additional AI components

### DISCUSSION THREAD

**Discussion by Dvalin21** [skrashevich/double-take#100] [Q&A] [answered]
Will there be other AI that will be added to Double-Take?

**Comment by skrashevich** [comment]
For example?..

**Reply by Dvalin21** [reply]
I know deepstack(Codeproject) is connected and Compreface, but what about darknet https://pjreddie.com/darknet/ or InsightFace https://github.com/SthPhoenix/InsightFace-REST?ref=hackernoon.com or DeepFace https://github.com/serengil/deepface?ref=hackernoon.com

**Comment by aniceberg** [comment]
Seconding this question... seems like both Deepstack and Facebox are both abandonware now. 

Can't even get the free API keys anymore to run them.

**Reply by skrashevich** [reply, marked as answer]
I'm ready to implement support for other recognition systems if they existed.
The above examples are not ready-made systems :(
